lib/src/helpers/enums.dart

```diff
@@ -5,9 +5,9 @@
 /// Helper layer library that exposes enums commonly used from `dart:html`.
 library;
 
-import '../../web.dart';
+import '../dom/html.dart';
 
-abstract final class KeyLocation {
```

Comment:
Why `sealed` over `abstract final`? These don't have constructors and we don't really need exhaustiveness (although I suppose `abstract sealed` doesn't hurt).